src/components/SlidingNavigation/SlidingNavigation.js

```diff
@@ -4,58 +4,60 @@ import classNames from "classnames";
 
 import "./SlidingNavigation.css";
 
-const SlidingNavigation = ({ data, options }) => {
+const SlidingNavigation = ({ data, options, isStorybook = false }) => {
   const activeItemClassName = `SlidingNavigation__item--active`;
   const observerOptions = options || {
     threshold: 0.2,
   };
 
   React.useEffect(() => {
-    const observer = new IntersectionObserver((entries) => {
-      entries.forEach((entry) => {
-        if (
-          entry.isIntersecting &&
-          !window.location.hash.includes(entry.target.id)
-        ) {
-          window.history.replaceState({}, ``, `#${entry.target.id}`);
-          const currentActiveElement = document.querySelector(
-            `.${activeItemClassName}`
-          );
+    if (!isStorybook) {
```

Comment:
:vomiting_face:

Reply:
@Toxicom `.SlidingNavigation` should be visible no matter what breakpoint it's in. It should become invisible if it's in a parent that renders a fallback navigation though.

The component is just a component. It's always visible and usable. Context may change it's behaviour and looks though.

Reply:
https://github.com/adaptdk/su-ukraina/pull/209/commits/57394454e23731b6b723f4733e5b30aaa20bba6c